clients/object_store_client.go

```diff
@@ -115,14 +115,25 @@ func PublishDriverSession(osUrl string, relPath string) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	videoUrl, err := osDriver.Publish(context.Background())
-	if err == drivers.ErrNotSupported {
-		// driver does not support Publish(), video will be accessible with osUrl
-		return osUrl, nil
-	} else if err != nil {
-		// error while publishing the video
-		return "", err
+
+	var videoUrl string
+	err = backoff.Retry(func() error {
```

Comment:
Should this exponential retry logic also be copied to `getFileHTTP`?

Reply:
I think that `getFileHTTP` has its own retry logic. But I'll update its parameters to be the same as here.